src/lib.rs

```diff
@@ -55,7 +55,23 @@ pub enum Link {
     Continue { id: u64 },
 }
 
-#[derive(Debug, Clone, Eq, PartialEq)]
+/// span id -> property
```

Comment:
Content in this PR description can put here

Reply:
Done